src/frontend/src/components/warnBox.ts

```diff
@@ -0,0 +1,36 @@
+import { html, TemplateResult } from "lit-html";
+import { warningIcon } from "./icons";
+
+interface warnBoxProps {
+  title: string;
+  message: string | TemplateResult;
+  componentType: string;
```

Comment:
shouldn't this be `"div" | "aside"`? Side note: do we really want to support both `aside` and `div`? CC @meodai

Reply:
Actually, having the warnBox **always** be an aside component makes sense to me. @meodai I think that may have been your intent as well? In your previous HTML semantics pull, you've changed a warnBox into an aside on this page [here](https://github.com/dfinity/internet-identity/blob/main/src/frontend/src/flows/addDevice/manage/index.ts#L59). But since the warnBox component was replicated many other places (as a div by default), I tried to include that option between div and an aside.

@nmattia 
I will make the change for the warnBox to be only an aside, because being encased in a div does not help with accessibility as you can see below:

Just as a div component: information looks fragmented in accessibility tree.

<img width="1000" alt="Screen Shot 2022-08-16 at 7 22 09 PM" src="https://user-images.githubusercontent.com/38238542/185021863-6868ce37-551d-4c8e-9706-a58fbb291ada.png">

As an aside component: the accessibility tree is much better.

<img width="1000" alt="Screen Shot 2022-08-16 at 7 20 53 PM" src="https://user-images.githubusercontent.com/38238542/185021875-4a44e6cc-be3a-47bc-88e1-35269b0b6876.png">

Reply:
I think its a good idea to pass in the `componentType` in some occasions, it will be annoying to wrap if in something else. I would just call it htmlElement or something like that and then interpolate it in the template instead of using an if statement.

`<${htmlElement}></${htmlElement}>`

`htmlElement: 'aside'`, is probably a good default though. ()

Reply:
> I think its a good idea to pass in the `componentType` in some occasions, it will be annoying to wrap if in something else. I would just call it htmlElement or something like that and then interpolate it in the template instead of using an if statement.
> 
> `<${htmlElement}></${htmlElement}>`
> 
> `htmlElement: 'aside'`, is probably a good default though. ()

I've given the option to wrap between aside (default) and a div. Unfortunately lit-html does not allow string interpolation inside a tag for its tag type. I think that's the best we can do, unless we explicitly specify another element type. 

<img width="450" alt="Screen Shot 2022-08-17 at 4 37 11 PM" src="https://user-images.githubusercontent.com/38238542/185261975-485cd051-d51d-481c-b78d-126813261c8d.png">